File: app/src/main/java/com/lusle/android/soon/Model/Schema/Videos.java

package com.lusle.android.soon.Model.Schema;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class Videos {

    @SerializedName("results")
    @Expose
    private ArrayList<Video> results = null;

    public ArrayList<Video> getResults() {
        return results;
    }

    public void setResults(ArrayList<Video> results) {
        this.results = results;
    }

}

File: app/src/main/java/com/lusle/android/soon/model/schema/Videos.kt
package com.lusle.android.soon.model.schema

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.*

data class Videos (
    @SerializedName("results")
    @Expose
    val results: ArrayList<Video>
) : Serializable{
}